src/ScriptBuilder.Tests/CecilExtensions/CustomAttributeMock.cs

```diff
@@ -32,10 +32,13 @@ TypeReference GetTypeReference(object value)
     public TypeReference AttributeType => throw new NotImplementedException();
     public bool HasFields => throw new NotImplementedException();
     public bool HasProperties => true;
+    public bool HasConstructorArguments => false;
     public Collection<CustomAttributeNamedArgument> Fields => throw new NotImplementedException();
 
     public Collection<CustomAttributeNamedArgument> Properties
     {
         get;
     }
+
+    public Collection<CustomAttributeArgument> ConstructorArguments => new Collection<CustomAttributeArgument>();
```

Comment:
How is this related to Newtonsoft update?

Reply:
Nope but we have a wildcard dependency on mono.cecil and they released a new breaking version.

Reply:
Would it make sense to lock in the version?

Reply:
I am not sure. We didn't have problems with this dependency so far.